src/enrichers/no-cookies-config.js

```diff
@@ -0,0 +1,12 @@
+import { isNonEmpty } from '../utils/types'
+
+export function enrich (state) {
+  if (isNonEmpty(state) && isNonEmpty(state.gdprApplies)) {
+    const gdprApplies = !!state.gdprApplies
+    return {
+      n3pc: gdprApplies,
```

Comment:
We also should skip computing context elements if gdprApplies.

Reply:
I propose to simplify the config enrichment part and only compute one field `privacyApplies`. Parameters sent to LivePixel then depend on this parameter and the same parameter can be used to detect if we need to skip computing context elements.